hikari/impl/rest.py

```diff
@@ -759,6 +772,23 @@ async def _request(
                     real_url = str(response.real_url)
                     raise errors.HTTPError(f"Expected JSON [{response.content_type=}, {real_url=}]")
 
+                # Handling 5xx errors
+                if response.status in _RETRY_ERROR_CODES and retries_done < self._max_retries:
+                    if backoff is None:
+                        backoff = rate_limits.ExponentialBackOff(maximum=16)
```

Comment:
Best not to hardcode this here. Create a global value ontop and use it here, similar to how shards use the ExponentialBackOff

Reply:
Understood